lib/common/socketpool.c

```diff
@@ -37,6 +37,7 @@ struct pool_entry_t {
     h2o_socket_export_t sockinfo;
     h2o_socketpool_target_t *target;
     h2o_linklist_t link;
```

Comment:
I would appreciate it if you could rename `link` to `all_link` so that the roles of the two links (the other is `target_link`) become clearer.